shards/modules/audio/audio.cpp

```diff
@@ -379,7 +379,7 @@ struct Channel {
     std::array<SHVar, 2> stereo;
     stereo[0] = Var(0);
     stereo[1] = Var(1);
-    _inChannels = Var(stereo);
```

Comment:
revert, this is already fixed no?

Reply:
Still good to change the default I think. up to you

Reply:
Undid the change